DataAccessLibrary/SqliteDataAccess.cs

```diff
@@ -50,7 +50,7 @@ public async Task<IEnumerable<(string, string, string, Dictionary<string, string
         public async Task<string> DbSave(string Table, string Column, string Value)
         {
             using IDbConnection connection = new SQLiteConnection(connectionString);
-            IEnumerable<string> output = await connection.QueryAsync<string>($"INSERT INTO {Table} ({Column}) VALUES ({Value})", new DynamicParameters());
+            IEnumerable<string> output = await connection.QueryAsync<string>($"INSERT INTO {Table} ({Column}) VALUES ({Value}) ON DUPLICATE KEY UPDATE ({Value})", new DynamicParameters());
 
             return "Data posted successfully";
```

Comment:
Det ville være nice, hvis den bare returnerede void, da vi aldrig vil vise brugeren dette.

Reply:
i will fix that.